packages/website/src/pages/layout/Section.tsx

```diff
@@ -1,241 +1,42 @@
-    Radio,
-    RadioSelect,
-    Section,
-    SingleSelectConnected,
-    Svg,
-    Tooltip,
-} from '@coveord/plasma-react';
+import {PageLayout} from '../../building-blocs/PageLayout';
 
-import PlasmaComponent from '../../building-blocs/PlasmaComponent';
+const code = `
+    import * as React from 'react';
+    import {Section} from '@coveord/plasma-react';
 
-// start-print
-export class SectionExamples extends React.Component<any, any> {
-    render() {
-        return (
-            <PlasmaComponent id="Section" title="Sections" withSource>
-                <Form>
-                    <Section
-                        title="Search panel"
-                        description="Customizing the display and behavior of the interface displayed withing the search panel can be done by editing the code of your search interface directly or via the JavaScript Search Interface Editor. As for style, it can be modified by applying your own stylesheet or adding styling rules in the Advanced tab."
-                    >
-                        <Section title="Main button">
-                            <LabeledInput
-                                label="Location on screen"
-                                helpText="The widget button's position is fixed, which means it is positioned relative to the viewport and stays in the same place even if the page is scrolled."
-                                optionalInformation="INFORMATION"
-                            >
-                                <RadioSelect>
-                                    <Radio value="1">
-                                        <Label>Value 1</Label>
-                                    </Radio>
-                                    <Radio value="2">
-                                        <Label>Value 2</Label>
-                                    </Radio>
-                                </RadioSelect>
-                            </LabeledInput>
-                            <Section title="Content">
-                                <LabeledInput helpText="The text that appears on the main widget button.">
-                                    <Input value="Help">
-                                        <Label>Text</Label>
-                                    </Input>
-                                </LabeledInput>
-                                <LabeledInput helpText="This is the font family that will be used on the button...">
-                                    <Input value="Canada type gibson, sans serif">
-                                        <Label>Font Family</Label>
-                                    </Input>
-                                </LabeledInput>
-                            </Section>
-                        </Section>
-                    </Section>
-                </Form>
+    export default () => (
+        <Section title="Section title" description="Section description.">
+            <Section title="Section title inside the anoter Section">
```

Comment:
Maybe remove this one from the basic demo and add a new one for nested sections?

Reply:
It doesn't really look like a child without the level=2 prop, also the child doesn't look weird to me 😛 

<img width="426" alt="image" src="https://user-images.githubusercontent.com/260007/158818077-92c7fe99-4cce-4926-9009-9678e4dac342.png">

Reply:
:joy:  It's looking weird on my cheap monitor :see_no_evil: